src/app/app.component.html

```diff
@@ -29,9 +29,18 @@ <h2 class="nunito-normal what-are-oeb">
 				Open Educational Badges?
 			</h2>
 		</div>
-		<video style="max-width: 900px; width: 75%; margin-top: 2em" height="auto" controls>
-			<source src="assets/videos/badges-curtain.mp4" type="video/mp4" />
-		</video>
+		<div
+			class="video-container md:tw-w-[52%] tw-h-[370px] tw-border-purple tw-border-solid tw-border-4 tw-m-4 tw-box-border"
+		>
+			<iframe
+				src="https://www.youtube.com/embed/H6lBdH75P7g?si=jiMGc0Jh1LDd2PrQ"
```

Comment:
The `?si=...` query parameter seems to be a tracker, so we should remove it from the link